modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -83,21 +113,52 @@ spec:
           name: cloud-credentials
         default: true
         objectStorage:
-          bucket: <bucket_name>
-          prefix: <bucket_prefix>
+          bucket: my-bucket
```

Comment:
Are we mandating that the bucket is always called `my-bucket` and that the prefix is always `velero`? The second makes some sense, but the first is questionable.  Remember even though this is a sample CR, we use <> to indicate where we expect the user to enter their own input. Please discuss this question with the QE reviewer for this PR.

Reply:
@PrasadJoshi12 @eemcmullan WDYT?

Reply:
The bucket and prefix name can be anything.